shared/reducers/crypto.tsx

```diff
@@ -36,6 +36,16 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
   [CryptoGen.resetStore]: () => {
     return initialState
   },
+  [CryptoGen.resetOperation]: (draftState, action) => {
+    const {operation} = action.payload
+
+    if (operationGuard(operation, action)) return
+
+    return {
```

Comment:
? why isn't this
`draftState[operation] = initialState[operation]`

Reply:
Typing got insane b.c. I couldn't convince TS that `draftState[operation] = initialState[operation]` were correct. There are two types for state. `CommonState` for `decrypt`, `verify`, and `sign`, and then `EncryptState` for `encrypt`.

Reply:
hmmmm. dunno

Reply:
If this is not so offensive, i'd like to leave it since it's a simple solution.